ark/utils/metacluster_remap_gui/.gitignore

```diff
@@ -0,0 +1,3 @@
+Pipfile
```

Comment:
Do we need a separate .gitignore here?

Reply:
I added this rule to ignore files that i personally used to create native venvs (in addition to docker venvs). This running allowed IDE autocompletion, linter rules, pytest, codecoverage, etc.

It's likely this gitignore rule is fine to add to the root `.gitignore`, but I was being conservative.

Additionally I know vscode IDE has support for connecting to docker container envs but it requires at least some configuration. Using a local venv just let me get to work quickly. I could,  work out and document method for running pytest locally via VSCode using the the docker venvs.

Reply:
That's okay, I think Adam already has this mostly worked out for his workflow

Reply:
Ok. I've removed folder specific .gitignore